recipes/dtw-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "dtw-python" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0b2448b66c32f61d3d78d0f03d5c37c6a2fb09e1e917c4cd662aff507059af83
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy
+    - setuptools
+  host:
+    - python
+    - pip
+    - numpy >=1.14
+    - setuptools
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - setuptools
```

Comment:
Do you actually rely on setuptools at runtime?

Reply:
AFAIK I need it because of `from pkg_resources import resource_string` to access data files in case they are zipped (build failed without it).

Reply:
Uggh that is terrible - no one should use pkg_resources for any reason

Reply:
I Will start looking for a fix when my anti- setup.py sentiment settles down... 😉

Reply:
yeah... just importing `pkg_resources` can cause python startup time to be >10 min on some systems. It is pretty bad